internal/uvm/create_lcow.go

```diff
@@ -659,13 +687,25 @@ func makeLCOWDoc(ctx context.Context, opts *OptionsLCOW, uvm *UtilityVM) (_ *hcs
 				return nil, err
 			}
 
-			doc.VirtualMachine.Devices.VirtualPci[vmbusGUID.String()] = hcsschema.VirtualPciDevice{
-				Functions: []hcsschema.VirtualPciFunction{
-					{
-						DeviceInstancePath: d.deviceInstanceID,
-						VirtualFunction:    d.virtualFunctionIndex,
+			if numa != nil || numaProcessors != nil {
```

Comment:
I think we could just do something like this:

```
if numa != nil || numaProcessors != nil {
    doc.VirtualMachine.Devices.VirtualPci[vmbusGUID.String()].PropagateNumaAffinity = true

}
```

Reply:
I had initially code it to what you suggested. But ran into this error:
golang cannot assign to struct field in map